src/main/java/magicbees/bees/MagicApiaryInventory.java

```diff
@@ -81,12 +81,11 @@ public void setInventorySlotContents(int i, ItemStack itemStack) {
     }
 
     public int[] getAccessibleSlotsFromSide(int side) {
-        if (side == 0 || side == 1) {
-            return new int[] { SLOT_QUEEN, SLOT_DRONE };
-        }
-        int[] slots = new int[SLOT_PRODUCTS_COUNT];
+        int[] slots = new int[SLOT_PRODUCTS_COUNT + 2];
+        slots[0] = SLOT_QUEEN;
+        slots[1] = SLOT_DRONE;
         for (int i = 0, slot = SLOT_PRODUCTS_START; i < SLOT_PRODUCTS_COUNT; ++i, ++slot) {
```

Comment:
`for (int i = 2, …`

Don't run unnecessary operations inside cycle.

Reply:
That's code that already existed

Reply:
See the change below this line.

He runs `i + 2` inside cycle, where he could just set `i = 2` initially and save yourself a few CPU cycles.

Reply:
That's a lot of noise over an omelette tbh